### PR TITLE
[WIP] Prepare next charter for WebRTC Working Group

### PR DESCRIPTION
**Pull request by dontcallmedom**

### CODE REVIEW

**Comment by dontcallmedom** [comment]
The last commit in this pull request tries to reflect our discussions on opening the path for media deliverables to migrate out of the WG

File: webrtc-charter.html [hunk 1]
```diff
@@ -294,24 +305,19 @@ <h3>
...
+          <p>The Working Group will maintain the following W3C Recommendations (gathering of errata, publication of new editions incorporating errata):</p>
+          <ul>
+            <li><a href="https://www.w3.org/TR/webrtc/">WebRTC 1.0: Real-time Communication Between Browsers</a></li>
+            <li><a href="https://www.w3.org/TR/mediacapture-streams/">Media Capture and Streams</a></li>
+          </ul>
 
-<p>As the name indicates, WebRTC 1.0: Real-time Communication Between Browsers is a first version of APIs for real-time communication, sometimes referred to as the PeerConnection API. The activities in the ORTC (Object Real-time Communications) Community Group indicate that there is interest in additional APIs to provide more direct control over WebRTC than what the PeerConnection API offers.</p>
+<p>The preferred mode of adding new features is by means of extension specifications, rather than new features in the existing specs. At appropriate times, the WG may choose to redistribute the work between chartered documents - by splitting, merging or repartitioning the work as appropriate.</p>
```
Comment (jan-ivar) [inline webrtc-charter.html:320]: Is this subject to change if W3C migrates to more of a live document model? Does this leave us an out?

File: webrtc-charter.html [hunk 2]
```diff
@@ -209,14 +214,6 @@ <h3>
...
           <dl>
-            <dt>Media Stream Functions</dt>
-            <dd>API functions to manipulate media streams for interactive real-time communications, connecting various processing functions to each other, and to media devices and network connections, including media manipulation functions for e.g. allowing to synchronize streams. Supplementary functions such as recording of media streams are also in scope.</dd>
-            <dt>Audio Stream Functions</dt>
-            <dd>An extension of the Media Stream Functions to process audio streams, to enable features such as automatic gain control, mute functions and echo cancellation.</dd>
-            <dt>Video Stream Functions</dt>
-            <dd>An extension of the Media Stream Functions to process video streams, to enable features such as bandwidth limiting, image manipulation or "video mute".</dd>
-            <dt>Functional Component Functions</dt>
-            <dd>API functions that allow to query for the components present in an implementation, instantiate them, and connect them to media streams.</dd>
             <dt>Data Access Functions</dt>
```
Comment (alvestrand) [inline webrtc-charter.html:217]: I think Data Access Functions for media streams belong with the Media functions.
We also have data access functions that belong in the comms part ("encoded data"), so perhaps both places?

File: webrtc-charter.html [hunk 3]
```diff
@@ -227,24 +224,45 @@ <h3>
...
           </dl>
 
+          <p>The Working Group will deliver specifications that cover the following functions until another group becomes available to take that work over:</p>
+          <dl>
+            <dt>Media Stream Functions</dt>
+            <dd>API functions to manipulate media streams for interactive real-time communications, connecting various processing functions to each other, and to media devices and network connections, including media manipulation functions for e.g. allowing to synchronize streams. Supplementary functions such as recording of media streams are also in scope.</dd>
+            <dt>Audio Stream Functions</dt>
+            <dd>An extension of the Media Stream Functions to process audio streams, to enable features such as automatic gain control, mute functions and echo cancellation.</dd>
+            <dt>Video Stream Functions</dt>
+            <dd>An extension of the Media Stream Functions to process video streams, to enable features such as bandwidth limiting, image manipulation or "video mute".</dd>
```
Comment (alvestrand) [inline webrtc-charter.html]: Bandwidth limiting is a comms function....

File: webrtc-charter.html (hunk 1 above)
Comment (alvestrand) [reply webrtc-charter.html:320]: If they adopt a living model, we can interpret "as appropriate" as "continuously".

File: webrtc-charter.html [hunk 4]
```diff
@@ -320,8 +326,7 @@ <h3>
 <li>Feature independence: Features may be introduced in the new APIs that are not available when using the PeerConnection API.</li>
 </ul>
 
-<p>A use-case document will be developed to clarify the functionality that the new APIs are to provide.</p>				
-<p>The Working Group will take the work done by the ORTC Community Group as a source of input, and when contemplating similar APIs in the Working Group, make efforts to align with the ORTC CG on API methodologies and nomenclature. This may include scheduled design meetings with relevant WG and CG stakeholders to foster convergence of the APIs.</p>
+<p>A <a href="https://www.w3.org/TR/webrtc-nv-use-cases/">use-case document</a> is maintained  to identify the functionality that the new APIs are to provide.</p>
```
Comment (alvestrand) [inline webrtc-charter.html]: would phrase this as "to identify the functionality that needs new APIs for realization".

**Comment by dontcallmedom** [comment]
@alvestrand pull request updated - please review

**Review by alvestrand — APPROVED** [review]

**Pull request merged**